subt/docker/robotika/Dockerfile

```diff
@@ -9,11 +9,31 @@ RUN /usr/bin/python3 -m pip install --user --no-cache-dir \
   "pyzmq==19.0.0"
 
 RUN cd /home/developer/ && head -n -2 .bashrc > .bashrc.new && mv .bashrc.new .bashrc
+RUN sudo apt update
```

Comment:
This should be part of line 23. See https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get:

> Always combine RUN apt-get update with apt-get install in the same RUN statement. Using apt-get update alone in a RUN statement causes caching issues and subsequent apt-get install instructions fail.

Reply:
ok